include/cxxopts.hpp

```diff
@@ -2300,6 +2300,58 @@ format_description
   return result;
 }
 
+String
+format_custom_help
+(
+  const std::string& custom_help,
+  std::size_t start,
+  std::size_t allowed
+)
+{
+  String result;
+  std::size_t count = 0;
+  std::size_t word_size = 0;
+
+  if(allowed <= start)
+  {
+    throw_or_mimic<exceptions::invalid_option_format>("Allowed column" 
+      "width must be greater than start column width!");
```

Comment:
(2) Typo
Missing concatanation of space in the error message
Should be

```suggestion
    throw_or_mimic<exceptions::invalid_option_format>("Allowed column " 
      "width must be greater than start column width!");
```